packages/desktop-client/src/components/budget/rollover/RolloverComponents.tsx

```diff
@@ -84,15 +85,29 @@ export function IncomeHeaderMonth() {
 }
 
 type ExpenseGroupMonthProps = {
+  month: string;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏sam here
```suggestion
  selectedMonth: string;
```

Reply:
See comment above.